db/Schema.dbml

```diff
@@ -1,57 +1,65 @@
 table organizations {
   id int [ pk, not null, increment ]
   name varchar [ not null, unique]
-  logo text [note: "Logo url"]
-  created_at timestamp [not null, default: `now()`]
+  logo text [default: null,note: "Logo url"]
+  created_at timestamp [ default: `now()`]
   updated_at timestamp [not null, default: `now()`, note: 'TODO: Add ON UPDATE ON UPDATE CURRENT_TIMESTAMP']
 }
 
+table knex_migrations_lock {
+  index int [ pk, not null, increment ]
+  is_locked int [ default: null]
+}
+
+table knex_migrations {
+  id int [ pk, not null, increment ]
+  name varchar [ default: null]
+  batch int [ default: null ]
+  migration_time timestamp [ default: null]
+}
+
```

Comment:
Sorry @AbodHassam, I did not think about this, but we do not want the Knex tables in our diagram. That's not really relevant to our app and just adding noise :)